packages/components/src/color-palette/index.js

```diff
@@ -112,19 +112,27 @@ function MultiplePalettes( {
 	);
 }
 
-export function CustomColorPickerDropdown( { isRenderedInSidebar, ...props } ) {
+export function CustomColorPickerDropdown( {
+	isRenderedInSidebar,
+	popoverProps: receivedPopoverProps,
+	...props
+} ) {
+	const popoverProps = {
+		__unstableShift: true,
+		// Open below the control point (centered aligned) when in the sidebar
+		...( isRenderedInSidebar
+			? {
+					placement: 'bottom',
+					offset: 10,
+			  }
+			: {} ),
+		...receivedPopoverProps,
```

Comment:
This change makes the CustomColorPickerDropdown for normal colors (non-gradients) appear differently.

Trunk:
![Screen Shot 2022-08-09 at 1 11 16 pm](https://user-images.githubusercontent.com/677833/183569868-6ef28e6b-84d8-47f3-8e7f-cab33be91cb5.png)

This branch:
![Screen Shot 2022-08-09 at 1 11 44 pm](https://user-images.githubusercontent.com/677833/183569915-dcd957a6-ac52-4397-869b-b14c24c8b544.png)

I can't see that discussed so far, the conversation has mostly been about the gradient picker.

@jasmussen Does this look ok, or should it stay to the left of the color palette dropdown?

Reply:
Since joen replied to this question in [his comment](https://github.com/WordPress/gutenberg/pull/42989#issuecomment-1209014992) below, I'll mark this convo as closed and will continue the conversation by replying directly to his comment